docs/manuals/source/TasksAndConcepts/Plugins/FileDaemonPlugins/VMwarePlugin.rst.inc

```diff
@@ -329,9 +330,18 @@ As of https://kb.vmware.com/s/article/2075984 manually enabling CBT is currently
                            disabled)
      --listall             List all VMs in the given datacenter with UUID and
                            containing folder
+     --sslverify           Force SSL certificate verification
+     --dumpvmconfig        Dump VM config metadata to JSON file
 
-Note: the options :command:`--vm-uuid` and :command:`--listall` have been added in version :sinceVersion:`17.2.8: VMware Plugin: new options in vmware\_cbt\_tool.py`, the tool is also able now to process non-ascii character arguments for the :command:`--folder` and :command:`--vmname` arguments and vApp names can be used like folder name components. With :command:`--listall` all VMs in the given datacenter are reported
-in a tabular output including instance UUID and containing Folder/vApp name.
+Note: the options :command:`--vm-uuid` and :command:`--listall` have been added in version :sinceVersion:`17.2.8: VMware Plugin: new options in vmware\_cbt\_tool.py`, the tool is also able now to process non-ascii character arguments for the :command:`--folder` and :command:`--vmname` arguments and vApp names can be used like folder name components.
```

Comment:
This note is not rendered as the other notes in the documentation , see this file line 287. Probably it makse sense to  format it like the other notes?

Reply:
Sure, it's formatted properly now.